tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "target": "ES6",
     "allowJs": true,
     "checkJs": true,
-    "lib": ["esnext.asynciterable"],
+    "lib": ["esnext.asynciterable","DOM"],
```

Comment:
There is an open issue with the latest version of `node-fetch` related to type definitions. You can track the discussion and updates on this issue at node-fetch/node-fetch#1617. AS a temp fix added DOMthe tsconfig

Reply:
Okay 👍 Can you please add a comment? Thanks!